pandas/core/indexes/multi.py

```diff
@@ -2441,7 +2442,53 @@ def reindex(self, target, method=None, level=None, limit=None, tolerance=None):
 
         return target, indexer
 
-    def get_slice_bound(self, label, side, kind):
+    def get_slice_bound(
+        self, label: Union[object, Tuple[object, ...]], side: str, kind: str
```

Comment:
```suggestion
        self, label: Union[Hashable, Sequence[Hashable]], side: str, kind: str
```

I think our labels should be a little more generic. Maybe worth putting something like this into pandas._typing @simonjayhawkins for thoughts